ipamd/ipamd.go

```diff
@@ -850,17 +929,25 @@ func getWarmIPTarget() int {
 	return noWarmIPTarget
 }
 
-func (c *IPAMContext) getCurWarmIPTarget() (int64, bool) {
+func (c *IPAMContext) ipTargetState() (short int, over int, enabled bool) {
```

Comment:
This now returns short (the number of IPs that we are short of the target), over (the number of IPs that we are over the target) and enabled.  I thought it made more sense than using a negative number to represent short, but maybe its more confusing.  Appreciate feedback.